### PR TITLE
Add British English translation

### PR DESCRIPTION
**Pull request by wjt**
We spell “colour” differently here.

### CODE REVIEW

File: po/en_GB.po [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the io.github.lainsce.Colorway package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Will Thompson <wjt@endlessos.org>, 2023.
+#
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/window.vala:591
```
Comment (wjt) [inline po/en_GB.po:8]: I just want to highlight something I also included in the commit message.

Note that marking the empty string for translation in window.vala as
follows:

            // TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example/'
            string translators = (_(""));

triggers a warning when generating the template, and based on the
message presumably does not work as intended at runtime:

    src/window.vala:591: warning: Empty msgid.  It is reserved by GNU gettext:
                                  gettext("") returns the header entry with
                                  meta information, not the empty string.

Comment (lainsce) [reply po/en_GB.po:8]: It works with the UI library being used, don't worry.

**Comment by lainsce** [comment]
Could you keep Colorway without the U? It's the app name so shouldn't be adapted/translated. :D

**Comment by wjt** [comment]
That's the whole point of the translation, for me! I can't cite any other apps as prior art but the film Upstream Color was released in the UK as Upstream Colour (with the original US audio). I can never find this app because desktop search for "colour" does not find it.

**Comment by lainsce** [comment]
It's mostly in the case of app brand. It's Colorway because it's a name, not the noun. Hope that helps.

**Comment by wjt** [comment]
I don't really agree – I see localising the spelling as more similar to transliterating the app name to a different alphabet (presumably it would be колорваи or similar in Ukrainian?) and I think having the app name spelled “wrong”, but the rest of the app spelled “correctly”, will look bad. I conducted a [highly scientific Mastodon poll](https://mastodon.me.uk/@wjt/110945558364298731) using a hypothetical app “Flavorsystem” and a very slim majority of my fellow Commonwealth English speakers agree with me. But, OK. I'll revise the PR.

**Comment by lainsce** [comment]
Thanks for the change, it's just a thing with a brand name, you don't see e.g. Apple becoming りんご in Japanese. 😅

**Comment by wjt** [comment]
No, but you do see the names of apps being localized when they contain functional terms, and you do on occasion see brands being transliterated.

If I set my phone language to French, “Amazon Shopping” becomes « Boutique Amazon », “Play Books” becomes « Play Livres » (though none of the other Play apps get localized), “Calendar” becomes « Agenda », “Digital Wellbeing” becomes « Bien-être numérique », (Google) “Translate” becomes « Traduction », etc.

I would set my phone to Ukrainian to see what gets transliterated to Cyrillic but after switching to French the system UI went into a crash loop so I'll stop here.

I guess my contention is that the word “Colorway” reads more like a functional term than a brand to me; and that changing it to “Colourway” is more like a transliteration. (And actually I have to say that the app drawer when I set my phone to French looks really bad, with 15% French words and 85% English words. Such is the privilege of English being my native language that I rarely get to experience this.)

Of course, it's your app and your call; and with the localised keywords I will be able to find this app now. Thanks for the merge!

**Pull request merged**